csi/controller.go

```diff
@@ -55,6 +55,15 @@ func (s *OsdCsiServer) ControllerGetCapabilities(
 		},
 	}
 
+	// Resizing volumes supported
+	capExpandVolume := &csi.ControllerServiceCapability{
+		Type: &csi.ControllerServiceCapability_Rpc{
+			Rpc: &csi.ControllerServiceCapability_RPC{
+				Type: csi.ControllerServiceCapability_RPC_EXPAND_VOLUME,
```

Comment:
We only need to implement `ControllerServiceCapability_RPC_EXPAND_VOLUME` as opposed to the Node Volume Expand capability, as our volumes can be resized while online (mounted)